hack/make-rules/helm.mk

```diff
@@ -1,67 +1,84 @@
-	$(ABSTOOLBIN)/helm install --dry-run ${HELM_RELEASE} ../${CHART} ${HELM_VALUES}
+	$(ABSTOOLBIN)/helm lint ${CHART_LOCAL_PATH}
+	$(ABSTOOLBIN)/helm install --dry-run ${HELM_RELEASE} ${CHART_LOCAL_PATH} ${HELM_VALUES}
 
 .PHONY: helm-uninstall
 helm-uninstall: $(TOOLBIN)/helm
 	$(ABSTOOLBIN)/helm uninstall ${HELM_RELEASE} || true
 
 .PHONY: helm-install
 helm-install: $(TOOLBIN)/helm
-	$(ABSTOOLBIN)/helm install ${HELM_RELEASE} ../${CHART} ${HELM_VALUES}
+	$(ABSTOOLBIN)/helm install ${HELM_RELEASE} ${CHART_LOCAL_PATH} ${HELM_VALUES}
+
 
+# example for helm chart push:
+# helm package fybrik-template -d /tmp/ --version 0.7.0
+# helm push /tmp/fybrik-template-0.7.0.tgz oci://localhost:5000/fybrik-system/
 .PHONY: helm-chart-push
-helm-chart-push: helm-login $(TOOLBIN)/helm
-	$(ABSTOOLBIN)/helm chart save ../${CHART} ${CHART_IMG}
-	$(ABSTOOLBIN)/helm chart list ../${CHART}
-	$(ABSTOOLBIN)/helm chart push ${CHART_IMG}
-	$(ABSTOOLBIN)/helm chart remove ${CHART_IMG}
+helm-chart-push: helm-login
+	$(ABSTOOLBIN)/helm package ${CHART_LOCAL_PATH} --version=${HELM_TAG} --destination=${TEMP}
+	$(ABSTOOLBIN)/helm push ${TEMP}/${CHART_NAME}-${HELM_TAG}.tgz ${CHART_REGISTRY_PATH}
+	rm -rf ${TEMP}/${CHART_NAME}-${HELM_TAG}.tgz
 
 .PHONY: helm-chart-pull
-helm-chart-pull: helm-login $(TOOLBIN)/helm
-	$(ABSTOOLBIN)/helm chart pull ${CHART_IMG} 
-	$(ABSTOOLBIN)/helm chart list
+helm-chart-pull: helm-login
+	$(ABSTOOLBIN)/helm pull ${CHART_REGISTRY_PATH}/${CHART_NAME} --version ${HELM_TAG}
 
-.PHONY: helm-chart-list
-helm-chart-list: $(TOOLBIN)/helm
-	$(ABSTOOLBIN)/helm chart list
+.PHONY: helm-list
+helm-list: $(TOOLBIN)/helm
+	$(ABSTOOLBIN)/helm list
 
 .PHONY: helm-chart-install
 helm-chart-install: $(TOOLBIN)/helm
-	$(ABSTOOLBIN)/helm chart export --destination=${TEMP} ${CHART_IMG} 
-	$(ABSTOOLBIN)/helm install ${HELM_RELEASE} ${TEMP}/${CHART} ${HELM_VALUES}
+	$(ABSTOOLBIN)/helm install ${HELM_RELEASE} ${CHART_REGISTRY_PATH}/${CHART_NAME} --version ${HELM_TAG} ${HELM_VALUES}
 	$(ABSTOOLBIN)/helm list
 
 .PHONY: helm-template
 helm-template: $(TOOLBIN)/helm
-	$(ABSTOOLBIN)/helm template ${HELM_RELEASE} ../${CHART} ${HELM_VALUES}
+	$(ABSTOOLBIN)/helm template ${HELM_RELEASE} ${CHART_REGISTRY_PATH} --version ${HELM_TAG} ${HELM_VALUES}
 
 .PHONY: helm-debug
-helm-debug: $(ABSTOOLBIN)/helm
-	$(ABSTOOLBIN)/helm template ${HELM_RELEASE} ../${CHART} ${HELM_VALUES} --debug
+helm-debug: $(TOOLBIN)/helm
+	$(ABSTOOLBIN)/helm template ${HELM_RELEASE} ${CHART_REGISTRY_PATH} ${HELM_VALUES} --version ${HELM_TAG} --debug
 
 .PHONY: helm-actions
-helm-actions:
-	$(ABSTOOLBIN)/helm show values ../${CHART} | yq -y -r .actions
+helm-actions: $(TOOLBIN)/helm
+	$(ABSTOOLBIN)/helm show values --version ${HELM_TAG}  ${CHART_REGISTRY_PATH} | yq -y -r .actions
 
 .PHONY: helm-all
-helm-all: helm-verify helm-chart-push helm-chart-pull helm-uninstall helm-chart-install
+helm-all: helm-verify helm-chart-push helm-chart-pull helm-chart-install helm-uninstall
```

Comment:
What is the purpose of the rule `helm-all` just check all other rules? maybe to rename it. 
otherwise, we install and after that uninstall the helm chart.

Reply:
I changed the name to `helm-check-all` to reflect it purpose.

Reply:
I fixed that. Thanks